processing/topics/bicycleParking/bicycleParking.lua

```diff
@@ -115,12 +120,13 @@ end
 
 function osm2pgsql.process_way(object)
   if exitProcessing(object) then return end
-  local tags = processTags(object.tags)
+  local result_tags = processTags(object.tags)
   local meta = Metadata(object)
+  meta.age = result_tags._age
```

Comment:
```suggestion
  local meta = Metadata(object, result_tags._age)
```

Wir haben das age doch auf allen Tabellen, richtig?
Dann fände ich es besser, es explizit zu übergeben statt indirekt zu setzen.

Dann kann man es auch auf meta = Metadata(…) (Ziel 130 hier) verschieben.

Als Parameter würde es explodieren, wenn der Param fehlt oder falsch ist. Als Zuweisung kann man es vergessen ohne Effekt.

Reply:
Nein, wir haben es nicht auf allen Tabellen. Zudem haben wir teilweise auch mehr als einen `*_age` tag, deshalb finde ich die manuelle Zusweisung hier passender.

Reply:
Wir könnten es allerdings  direkt in der `Metadata` Funktion mit erzeugen, falls wir das wollen.